std/digest/murmurhash.d

```diff
@@ -0,0 +1,815 @@
+/**
+Computes MurmurHash hashes of arbitrary data. MurmurHash is a non-cryptographic
+hash function suitable for general hash-based lookup.
+
+This module conforms to the APIs defined in $(D std.digest.digest).
+
+This module publicly imports $(D std.digest.digest) and can be used as a stand-alone module.
+
+Note: The current implementation is optimized for little endian architectures.
+It will exhibit different results on big endian architectures and a slightly less uniform distribution.
+
+License: $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
```

Comment:
@WalterBright is this ok? (I borrowed it from another file)
BTW it looks like not all files have copyright.